contributors/devel/sig-testing/e2e-tests.md

```diff
@@ -604,6 +604,32 @@ Note that advanced testing parameters, and hierarchichally defined parameters, a
 
 In time, it is our intent to add or autogenerate a sample viper configuration that includes all e2e parameters, to ship with Kubernetes.
 
+### Pod Security Admission
+
+With introducing Pod Security admission in Kubernetes by default, it is desired to execute e2e tests within bounded pod security policy levels. The default pod security policy in e2e tests is [restricted](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-levels). This ensures that e2e tests follow best practices for hardening pods by default.
+
+Two helper functions are available for returning a minimal [restricted pod security context](https://github.com/kubernetes/kubernetes/blob/d7e6eab87d0fd005b238e3ec9b088e37d41a15d3/test/e2e/framework/pod/utils.go#L119) and a [restricted container security context](https://github.com/kubernetes/kubernetes/blob/d7e6eab87d0fd005b238e3ec9b088e37d41a15d3/test/e2e/framework/pod/utils.go#L127). These can be used to initialize pod or container specs to ensure adherence for the most restricted pod security policy.
+
+If pods need to elevate privileges to either `baseline` or `privileged` a new field was introduced to the e2e framework to specify the necessary namespace enforcement level:
```

Comment:
Specifically name the field so that it's clear which one it is.
```suggestion
If pods need to elevate privileges to either `baseline` or `privileged` a new field - `NamespacePodSecurityEnforceLevel` - was introduced to the e2e framework to specify the necessary namespace enforcement level:
```

Reply:
Maybe also point out that the namespace gets created in the framewor's `BeforeEach()` procedure to point out when this field needs to be set.

Reply:
fixed